### PR TITLE
Updated to refer to using relational databases for DR

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated to refer to using relational databases for DR

### CODE REVIEW

File: docs/Configuring-Orion/Disaster-Recovery.md [hunk 1]
```diff
@@ -4,18 +4,15 @@ description: Disaster recovery strategies
 # Disaster Recovery
 
 !!! danger
-    If the database is deleted or corrupted, all payloads for the node are lost. You cannot recover a lost database without a backup.
+    If the Orion database is deleted or corrupted, all private transaction payloads for the node are lost. 
+    You cannot recover a lost database without a backup.
 
-Orion stores all payload information in an internal database. The database location is specified by the 
-`workdir` and `storage` properties in the [configuration file](Configuration-File.md).
+Orion supports using relational databases PostgreSQL and Oracle. To enable disaster recovery for Orion,
```
Comment (joshuafernandes) [inline docs/Configuring-Orion/Disaster-Recovery.md:10]: Maybe add more are planned on the roadmap?

File: docs/Configuring-Orion/Disaster-Recovery.md [hunk 2]
```diff
@@ -4,18 +4,15 @@ description: Disaster recovery strategies
...
-```
-
-The level db database is located in `/orionNode/data`
-
-When using Orion in production, we recommend that you have a backup strategy and a data redundancy setup.
+To recover after a failure:
 
+1. Start Orion pointing to your database
+1. Start Hyperledger Besu. As Besu synchronizes to the current world state it [processes the privacy marker 
+transactions](https://besu.hyperledger.org/en/stable/Concepts/Privacy/Private-Transaction-Processing/)
```
Comment (joshuafernandes) [inline docs/Configuring-Orion/Disaster-Recovery.md:17]: :+1:

**Review by joshuafernandes — APPROVED** [review]
LGTM

File: docs/Configuring-Orion/Disaster-Recovery.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Configuring-Orion/Disaster-Recovery.md:10]: We don't do this in the doc generally unless there's a really strong reason to (ie, there's a big limitation that we want to highlight is about to be fixed or if a choice they make now will limit their choices in regards to a planned feature)

**Pull request merged**